DSharpPlus/Entities/AutoModeration/Action/DiscordRuleActionMetadata.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+
+using Newtonsoft.Json;
+
+namespace DSharpPlus.Entities;
+
+/// <summary>
+/// Represents a Discord rule action metadata.
+/// </summary>
+public class DiscordRuleActionMetadata
+{
+    /// <summary>
+    /// Gets the channel which the blocked content should be logged.
+    /// </summary>
+    [JsonProperty("channel_id")]
+    public ulong ChannelId { get; internal set; }
+
+    /// <summary>
+    /// Gets the timeout duration in seconds.
+    /// </summary>
+    [JsonProperty("duration_seconds")]
+    public uint TimeoutDuration { get; internal set; }
```

Comment:
It would be desirable to rename this to `TimeoutSeconds` or expose a `TimeSpan` that parses from this field (making this private).